cmd/terramate/cli/cli.go

```diff
@@ -405,6 +405,50 @@ func (c *cli) generate() {
 	}
 }
 
+func (c *cli) checkGitUntracked() bool {
+	if c.parsedArgs.DisableCheckGitUntracked {
+		return false
+	}
+
+	if disableCheck, ok := os.LookupEnv("TM_DISABLE_CHECK_GIT_UNTRACKED"); ok {
+		if envVarIsSet(disableCheck) {
+			return false
+		}
+	}
+
+	cfg := c.prj.rootcfg
+
+	if cfg.Terramate != nil &&
+		cfg.Terramate.Config != nil &&
+		cfg.Terramate.Config.Git != nil {
```

Comment:
we should improve those nil checks later...Now that we have default values in the config, maybe it doesn't need to be a pointer anymore. But we can improve this later.

Reply:
👍